src/main/java/duke/Events.java

```diff
@@ -0,0 +1,38 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * This class is implementation of Task which contains description, from and to.
+ */
+public class Events extends Task {
```

Comment:
Similar to my comment on your `Todos` class, I feel like this class should be named `Event` instead of `Events`.